packages/contracts/scripts/smt-checker/README.md

```diff
@@ -12,7 +12,7 @@ Ensure that your system has Z3 installed with a version number greater than or e
 Before proceeding, ensure that you have `g++` and `python3` installed on your system. To download and install Z3, enter the following commands in your terminal:
 
 ### Run script:
-```
+```sh
 sh ./update-dependencies.sh
```

Comment:
The scripts were renamed:
```
./check-deps.sh
./update-deps-ubuntu.sh
```
It should also reflect the README content. Also sh can be removed, as the shell scripts have executable bit set and can be executed directly from shell:
```
$ ./check-deps.sh 
All required programs are installed on your system.
$ ./update-deps-ubuntu.sh 
```

Reply:
indeed